app.js

```diff
@@ -1,15 +1,24 @@
 const express = require("express");
-const { getCategoriesC } = require("./controllers/controllers");
+const { getCategoriesC, getReviewsByIdC } = require("./controllers/controllers");
 const error_404_msg = "Sorry can't find that!";
```

Comment:
do these need to exist globally? I think that it would be cleaner to have them inside your error handling middleware